libs/llrb/llrb_test.go

```diff
@@ -54,7 +54,10 @@ func TestBasics(t *testing.T) {
 	if tree.Size() != 1 {
 		t.Errorf("expecting len 1")
 	}
-	data, _ := tree.Remove(*nks[0])
+	data, err := tree.Remove(*nks[0])
+	if err != nil {
```

Comment:
can use `assert` / `require` libraries for result check. you can find their usages in other tests